### DISCUSSION TITLE
PHP & DB errors (user_sota_look & user_show_notes) after git pull

### DISCUSSION THREAD

**Discussion by minorsecond** [magicbug/Cloudlog#975] [Q&A] [answered]
So I just pulled the repository and was met with two errors after pulling up the webpage:

```
Message: Undefined property: stdClass::$user_sota_lookup
Filename: models/User_model.php
Line Number: 267

Message: Undefined property: stdClass::$user_show_notes
Filename: models/User_model.php
Line Number: 268
```

I figured that new options had been added, so I checked the user profile settings. Sure enough, there is a new SOTA auto look option, and a new "Show notes" option. I thought that re-saving my user config might add those options to my configuration, but no dice. After saving, I get a database error:

```
Unknown column 'user_sota_lookup' in 'field list'

UPDATE `users` SET `user_name` = '***', `user_email` = '***@gmail.com', `user_callsign` = 'KD5LPB', `user_locator` = 'DM79po', `user_firstname` = '***', `user_lastname` = '***', `user_timezone` = '84', `user_lotw_name` = '***', `user_eqsl_name` = '', `user_clublog_name` = '', `user_measurement_base` = 'M', `user_date_format` = 'Y-m-d', `user_stylesheet` = 'default', `user_sota_lookup` = '0', `user_show_notes` = '0', `user_type` = '99' WHERE `user_id` = '6'

Filename: models/User_model.php

Line Number: 202
```

should I go in and add the user_sota_lookup column to the users table? Will there be more to this? Where is the DB file?

I see in the migrations directory, there are two files:
`064_add_user_sota_lookup.php` and `066_add_user_hide_notes.php`. Did these not get called for some reason?

**Comment by AndreasK79** [comment, marked as answer]
It seems to me that those have not been called yes. They should normally be called when entering the log.
I have no idea why, but maybe @magicbug have an idea?

You can add these manually if you want.
`alter table users add user_sota_lookup integer DEFAULT 0`
`alter table users adduser_show_notes integer DEFAULT 1`

**Reply by minorsecond** [reply]
Thanks. Where is the database located? I assume it's sqlite?

**Reply by AndreasK79** [reply]
This might be related to a migration script I added, but with a wrong name. If you can try to rename 067_aidd_index_cqz_prop.php to 067_add_index_cqz_prop.php in the migrations folder. Then reload Cloudlog (F5). I hope that might fix your problem. I've added a pull request to fix this.

You can also run "#cloudlogurl#/index.php/migrate" to update to the latest scripts after renaming the file above.

And to answer your question, it is MySQL. If you're not set up to run SQL commands, it might not be so easy to add these yourself.

**Reply by minorsecond** [reply]
That worked! Thanks!

**Reply by magicbug** [reply]
Fixed in the codebase too.
